CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## TBD
+### Added
+- `apiary-receivers-parent` parent module and `apiary-receiver-sqs` sub-module, a library to poll Apiary SQS messaging infrastructure for Hive events.
```

Comment:
The module should be called `apiary-receivers`, it's just the pom at the top of that module which will have the artifactId of `apiary-receivers-parent` but nothing outside of the poms referring to it need to know that. That's the same pattern we have for this project - the top-level module is called `apiary-extensions` while the top-level pom is called `apiary-extensions-parent`;

Reply:
Ah I see now, will make this change. Thanks!

Reply:
Done